app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -465,6 +460,17 @@ class NewMessageFragment : Fragment() {
         newMessageViewModel.uiBodyLiveData.observe(viewLifecycleOwner, bodyTextField::setText)
     }
 
+    private fun observeUiQuote() = with(binding) {
+        newMessageViewModel.uiQuoteLiveData.observe(viewLifecycleOwner) { quote ->
+            if (quote == null) {
+                trackNewMessageEvent("deleteQuote")
```

Comment:
the tracking should be in the clickListener (we track the user intention, not the programming updates it triggers)